puppet/zulip/manifests/postgresql_base.pp

```diff
@@ -75,15 +75,20 @@
   $pgroonga = zulipconf('machine', 'pgroonga', '')
   if $pgroonga == 'enabled' {
     # Needed for optional our full text search system
+
     package{"${postgresql}-pgroonga":
+      ensure  => 'purged',
+    }
```

Comment:
Can we accompany this with a comment as to when we can remove these lines?  Something like:
```
# Removed 2020-12 in version 4.0; these lines can be removed in Zulip version 5.0 and later
```
...so that we don't persist this upgrade step in perpetuity.

Reply:
Done.
We can remove these lines when we drop support for upgrading from Zuilp 3 or older.